### PR TITLE
Recommend python 3.6 for Manual Installation

### PR DESCRIPTION
**Pull request by gherceg**
Since we plan to no longer support Python 2 in the very near future this should recommend to use Python 3 (specifically 3.6). Feel free to improve wording.
##### ENVIRONMENTS AFFECTED

None

### CODE REVIEW

**Review by millerdev — COMMENTED** [review]
There are a few other things that should be updated. I'll add them to this PR

**Comment by gherceg** [comment]
Thanks @millerdev!

File: docs/setup/installation.md [hunk 1]
```diff
@@ -55,6 +55,21 @@ then the `git clone` command did not run correctly.
 Make sure you have git installed and run it again
 with `--verbose` to give more logging output.
 
+If you see
+
+```bash
+fatal: destination path 'commcare-cloud' already exists and is not an empty directory.
```
Comment (dannyroberts) [inline docs/setup/installation.md:61]: Just kind of curious, I've never seen this. When does/can it happen?

File: control/check_install.sh [hunk 2]
```diff
@@ -39,51 +39,9 @@ else
...
-            ln -sf $(which ${executable}) ~/.commcare-cloud/bin/
-            printf "${YELLOW}→ Created ~/.commcare-cloud/bin/${executable}\n"
-        fi
-    else
-        printf "${GREEN}✓ ~/.commcare-cloud/bin/${executable} exists\n"
-    fi
-done
+# clean up obsolete ~/.commcare-cloud stuff
+[ -d ~/.commcare-cloud/bin ] && { rm ~/.commcare-cloud/bin/*; rmdir ~/.commcare-cloud/bin }
+[ -h ~/.commcare-cloud/repo ] && rm ~/.commcare-cloud/repo
```
Comment (dannyroberts) [inline control/check_install.sh:44]: Did you know that `manage-commcare-cloud` also references / uses / sets stuff up in `~/.commcare-cloud`? https://github.com/dimagi/commcare-cloud/blob/d6ac1a4172a2e67927d80c393136d9900ebecdd9/src/commcare_cloud/manage_commcare_cloud/configure.py#L80-L83. I think it doesn't overlap with this stuff, but I didn't read super closely. Just wanted to make sure that was on your radar

**Review by dannyroberts — COMMENTED** [review]
Looks generally good assuming it the conversion works seamlessly. Had a couple questions

File: docs/setup/installation.md (hunk 1 above)
Comment (millerdev) [reply docs/setup/installation.md:61]:
```sh
$ mkdir commcare-cloud
$ touch commcare-cloud/.git
$ git clone git@github.com:dimagi/commcare-cloud.git
fatal: destination path 'commcare-cloud' already exists and is not an empty directory.
```

File: control/check_install.sh (hunk 2 above)
Comment (millerdev) [reply control/check_install.sh:44]: Yes, I was originally intending to remove that too, but realized that it's an orthogonal set of stuff and decided I didn't want to expand the scope of this PR to include it.

**Comment by millerdev** [comment]
@dannyroberts is there a reason why you didn't approve?

**Review by dannyroberts — APPROVED** [review]
Nope no reason. Was just waiting to understand, but looks good

**Pull request merged**